docs/_includes/Common-Commands/download-alluxio.md

```diff
@@ -1,4 +1,4 @@
 ```bash
-wget http://alluxio.org/downloads/files/{{site.ALLUXIO_RELEASED_VERSION}}/alluxio-{{site.ALLUXIO_RELEASED_VERSION}}-bin.tar.gz
+wget http://alluxio.io/downloads/files/{{site.ALLUXIO_RELEASED_VERSION}}/alluxio-{{site.ALLUXIO_RELEASED_VERSION}}-bin.tar.gz
```

Comment:
alluxio.io/downloads/files/... only points people to the downloads or release notes pages. for wget, use http://downloads.alluxio.io/downloads/files/...
 
http://downloads.alluxio.io/downloads/files/{{site.ALLUXIO_RELEASED_VERSION}}/alluxio-{{site.ALLUXIO_RELEASED_VERSION}}-bin.tar.gz

Reply:
done